### ISSUE TITLE
Validation Method

### ISSUE THREAD

**Issue by bajarno** [ZJULearning/pixel_link#123]
What validation method is used to select the model for which results are reported on the ICDAR2015 test set?

I was unable to find how a model is selected while reading the paper and looking through the source code. It seems like the complete training set is used for training, so no data is left for a separate validation set.

I am trying to reproduce the results using PyTorch. My model is performing reasonably well, but the results are rather unstable over different epochs. Some validation method is needed to properly select a good model and experiment with hyperparameters while preventing overfitting on the test set.

Thank you!
